src/bridges/erc4626/VaultBridge.sol

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Copyright 2020 Spilsbury Holdings Ltd
+pragma solidity ^0.8.4;
+pragma experimental ABIEncoderV2;
```

Comment:
ABIEncoderV2 is default and no longer experimental so please remove the line above.